### PR TITLE
update doc for kubelet option --reserved-cpus

### PR DESCRIPTION
**Pull request by jianzzha**
This website update is for the new kubelet CLI option: https://github.com/kubernetes/kubernetes/pull/83592

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -1024,6 +1024,13 @@ kubelet [flags]
       <td></td><td style="line-height: 130%; word-wrap: break-word;">If > 0, limit registry pull QPS to this value.  If 0, unlimited. (default 5) (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
     </tr>
 
+     <tr>
+        <td colspan="2">--reserved-cpus string</td>
+     </tr>
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: Hello @jianzzha . You should make this change in the kubelet cmd line code (k8s/k8s).

Comment (jianzzha) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: @kbhawkey thanks,  this file is reverted.

File: content/en/docs/tasks/administer-cluster/cpu-management-policies.md [hunk 2]
```diff
@@ -85,7 +85,10 @@ state file `cpu_manager_state` in the kubelet root directory.
 This policy manages a shared pool of CPUs that initially contains all CPUs in the
 node. The amount of exclusively allocatable CPUs is equal to the total
 number of CPUs in the node minus any CPU reservations by the kubelet `--kube-reserved` or
-`--system-reserved` options. CPUs reserved by these options are taken, in
+`--system-reserved` options. From 1.17, the CPU reservation list can be specified 
+explictly by kubelet --resevered-cpus option. The explicit CPU list specified by 
+--resevered-cpus will take precedence over the CPU reservation specified by
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/cpu-management-policies.md]: two typos: resevered -> reserved

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 3]
```diff
@@ -146,6 +146,28 @@ control group (`system.slice` on systemd machines for example).
...
+`reserved-cpus` is meant to define an explict CPU set for OS system daemons and 
+kubernetes system daemons. This option is added in 1.17 release. `reserved-cpus` 
+is for systems that do not intent to define seperate top level cgroups for 
+OS system daemons and kubernetes system daemons with regard to cpuset resource. 
+If the Kubelet **does not** have `--system-reserved-cgroup` and `--kube-reserved-cgroup`, 
+the explicit cpuset provided by `reserved-cpus` will take precedence over the CPUs 
+defined by `--kube-reserved` and `--system-reserved` options.
+
+This option is specifically designed for Telco/NFV use cases where uncontrolled 
+interrupts/timers may impact the work load performance. Users may use this option
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md]: work load -> workload

**Comment by sftim** [comment]
Did you want this PR to target branch _dev-1.17_ instead BTW?

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 4]
```diff
@@ -146,6 +146,28 @@ control group (`system.slice` on systemd machines for example).
 Note that Kubelet **does not** create `--system-reserved-cgroup` if it doesn't
 exist. Kubelet will fail if an invalid cgroup is specified.
 
+### Explictly Reserved CPU List
+
+- **Kubelet Flag**: `--reserved-cpus=`
+
+
+`reserved-cpus` is meant to define an explict CPU set for OS system daemons and 
+kubernetes system daemons. This option is added in 1.17 release. `reserved-cpus`
```
[the remark this answers is not in the document]
Comment (jianzzha) [reply content/en/docs/tasks/administer-cluster/reserve-compute-resources.md]: added. check if it is what's expected

**Comment by jianzzha** [comment]
> Did you want this PR to target branch _dev-1.17_ instead BTW?
yes. I couldn't add that tag myself

**Comment by tengqm** [comment]
As pointed out by @sftim, this PR is about a feature to be landed in 1.17, right?
Please rebase onto dev-1.17 branch if so.
/hold

**Comment by jianzzha** [comment]
> As pointed out by @sftim, this PR is about a feature to be landed in 1.17, right?
> Please rebase onto dev-1.17 branch if so.
> /hold

@tengqm yes. how to do the rebase on 1.17? any instructions?

**Comment by tengqm** [comment]
@jianzzha You may want to check this: https://git-scm.com/book/en/v2/Git-Branching-Rebasing

Basically, the commands are:

```
$ git checkout <your-pr-branch>
$ git rebase dev-1.17
First, rewinding head to replay your work on top of it...
Applying: added staged command
$ git commit --amend --no-edit
$ git push -f
```

**Comment by jianzzha** [comment]
@tengqm @sftim please review https://github.com/kubernetes/website/pull/17648 in stead - that PR is base on dev-1.17. I will close this PR. thanks!

**Comment by jianzzha** [comment]
@tengqm @sftim please review #17648; it captured all the changes from this PR and it is based off  dev-1.17.

**Pull request closed without merging**